sormas-api/src/main/resources/captions.properties

```diff
@@ -1092,7 +1092,7 @@ Exposure.prophylaxis=Post-exposure prophylaxis
 Exposure.prophylaxisDate=Date of prophylaxis
 Exposure.riskArea=Risk area as defined by public health institution
 Exposure.exposureDate=Exposure date
-Exposure.patientExpositionRole=In the role of
+Exposure.exposureRole=In the role of
```

Comment:
Change to just "Role" and remove the line from all other property files (because it's technically a new field with a new caption then and should be re-translated)

Reply:
done